semgrep_output_v1.atd

```diff
@@ -89,14 +89,21 @@ type location
 }
 
 (*****************************************************************************)
-(* Rule id *)
+(* Rules *)
 (*****************************************************************************)
 
 (* e.g., "javascript.security.do-not-use-eval" *)
 type rule_id
   <ocaml attr="deriving show">
   <python decorator="dataclass(frozen=True)"> = string
 
+type rule
```

Comment:
Please do not use rule. Rule means something special in Semgrep (see Rule.ml) where
we have a rule id, a message, a formula, etc.
I'd rather use rule_id_and_engine_kind